src/components/PluginCapabilitiesSection.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <p class="text-3xl py-4" id="capabilities">Capabilities</p>
-    <span class="space-y-3" v-if="capabilities">
+    <span class="space-y-3" v-if="len(capabilities) > 0">
```

Comment:
```suggestion
    <span class="space-y-3" v-if="capabilities && capabilities.length">
```

Based on how settings is done I think this is the way?